codice/ddf
# DDF-5373 Remove manual metacard creation view

#### What does this PR do?
Removes the ability for users to manually create metacards to be added to their lists from within the workspaces view. 
This feature was being used by downstream projects but is no longer desired. 

#### Who is reviewing it? 

@gordocanchola 
@andrewzimmer 
@willwill96 
@hayleynorton 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining
@djblue

#### How should this be tested?

Build
Profile:install
Create a list and click the button circled in the first screenshot below. 
Verify that the modal that opens has only one tab, `import`, and no tab called `manual`, as shown in the second screenshot.

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: #5373 

#### Screenshots

![button](https://user-images.githubusercontent.com/14204943/65913980-fb72a800-e38d-11e9-9599-3414a7966a95.PNG)
![modal](https://user-images.githubusercontent.com/14204943/65913981-fb72a800-e38d-11e9-8566-034de6ccb8d9.PNG)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
